### ISSUE TITLE
Strange autocomplete

### ISSUE THREAD

**Issue by iliakan** [lukehaas/RunJS#300]
Why `userInfo` ?
![image](https://user-images.githubusercontent.com/349336/128676691-53bbbf1b-cfe3-4648-9932-6205d3c45802.png)

**Comment by lukehaas** [comment]
Suggestions are just coming straight from the typescript language service. They're not something the app controls.
userInfo is also present in VSCode suggestions:
![Screenshot 2021-08-09 at 09 40 22](https://user-images.githubusercontent.com/741344/128679989-31143ec9-ae93-42f5-9245-577a3ea32072.png)

**Comment by iliakan** [comment]
Your VSCode suggestions are unlike mine.

![image](https://user-images.githubusercontent.com/349336/128680330-c43266e0-59e4-4586-b1f4-ce45202887cb.png)

It must be possible to turn this "super smart" (ugly) machinery off. 

RunJS is meant for small scripts, not big projects.

**Comment by iliakan** [comment]
P.S. Many people use just JS, and don't want any TypeScript-related stuff ;). Just FYI =)

**Comment by iliakan** [comment]
I mean, make autocomplete only list values from the *current* script.

**Comment by iliakan** [comment]
Or maybe from another tabs, from standard JS, but not any more.

**Comment by lukehaas** [comment]
If a language service had the ability to do that, I'd be happy to support it.

**Comment by iliakan** [comment]
Maybe it has some parameters? What exactly do you use (I'll look it up)?

**Comment by lukehaas** [comment]
Sure: https://github.com/Microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29

**Comment by iliakan** [comment]
So, there's `tsconfig.json` where you supply all parameters, including those that affect autocomplete, right?

**Comment by lukehaas** [comment]
That's right. Just to clarify, what is it exactly you want to see and not see in the autocomplete?

**Comment by iliakan** [comment]
I'd like to see variables from the current tab only.
Sometimes standard JS names can be ok too, but not DOM-related (as RunJS has no DOM).

Plain JS-only not overly smart autocomplete ;)

**Comment by lukehaas** [comment]
I see. Only issue there for me is for the DOM, as RunJS does have DOM access.

**Comment by iliakan** [comment]
Oh, fine, let it keep DOM then.

`alert` and `prompt` (BOM) not working though =(

Personally, I think `prompt` might be helpful sometimes =)

**Comment by lukehaas** [comment]
Yeah, `alert` and `prompt` are not supported. Also `print` is not supported.
